code/modules/unit_tests/slime_mood.dm

```diff
@@ -0,0 +1,11 @@
+///Unit test that tests all types of moods for slimes, to make sure they all have proper icons, excluding moods that intentionally don't have an icon.
+/datum/unit_test/slime_mood
+
+/datum/unit_test/slime_mood/Run()
+	var/mob/living/simple_animal/slime/this_guy = allocate(/mob/living/simple_animal/slime)
```

Comment:
This honest to god is not better than `S`

Reply:
Get a load of this guy
I've seen a lot of funny names on allocates mobs in unit test and thought why not do the same